### ISSUE TITLE
EIP4844: Remove redundant cryptography point in networking specs

### ISSUE THREAD

**Issue by kevaundray** [ethereum/consensus-specs#3137]
**Motivation**

After changing the cryptography to be more higher level, certain parts of the spec now are no longer concerned with particular cryptographic invariants, since they are encapsulated inside of the cryptography module.

It seems that some parts of the spec were not updated to reflect this. In particular in [networking](https://github.com/ethereum/consensus-specs/blob/a1e46d1ae47dd9d097725801575b46907c12a1f8/specs/eip4844/p2p-interface.md#beacon_block_and_blobs_sidecar):

- [REJECT] the sidecar.blobs are all well formatted, i.e. the BLSFieldElement in valid range (x < BLS_MODULUS).

- [REJECT] The KZG proof is a correctly encoded compressed BLS G1 Point -- i.e. bls.KeyValidate(blobs_sidecar.kzg_aggregated_proof)

**Suggestion**

Since:

- [REJECT] The KZG commitments in the block are valid against the provided blobs sidecar. -- i.e. validate_blobs_sidecar(block.slot, hash_tree_root(block), block.body.blob_kzg_commitments, sidecar)

Will only pass if the blobs are all well formatted ie canonical and the proof point is correctly encoded, those two points now seem to be redundant.

**Notes**

If the above is accepted, then I would also question if the networking layer needs this point:

- [REJECT] The KZG commitments of the blobs are all correctly encoded compressed BLS G1 Points. -- i.e. all(bls.KeyValidate(commitment) for commitment in block.body.blob_kzg_commitments)

In order to remove it, I think we would need to add it inside of `verify_kzg_commitments_against_transactions` . If we always validate the commitments before calling `verify_kzg_commitments_against_transactions` then I think it would be reasonable to move the point validation function inside of `verify_kzg_commitments_against_transactions`.

**Comment by dankrad** [comment]
Yeah I agree with that. We can leave all the validation to the cryptography library.

**Comment by leolara** [comment]
I am closing this issue because it seems stale. Please, do not hesitate to reopen it if this is a mistake
